integration/src/main/java/scripts/utils/CloudContextMaker.java

```diff
@@ -36,26 +35,6 @@ public static String createGcpCloudContext(UUID workspaceUuid, WorkspaceApi work
     return contextResult.getGcpContext().getProjectId();
   }
 
-  /**
-   * Creates an Azure cloud context for a given workspace. Returns the Azure resource group ID as a
-   * string
-   */
-  public static String createAzureCloudContext(
```

Comment:
Method is now unused after https://github.com/DataBiosphere/terra-workspace-manager/pull/1166.